framework/PageDetails/PageDetail.tsx

```diff
@@ -31,9 +32,13 @@ export function PageDetail(props: {
           {helpText ? <StandardPopover header={label} content={helpText} /> : null}
         </DescriptionListTerm>
       )}
-      <DescriptionListDescription id={id} data-cy={id}>
+      <DescriptionListDescriptionStyled id={id} data-cy={id}>
         {children}
-      </DescriptionListDescription>
+      </DescriptionListDescriptionStyled>
     </DescriptionListGroup>
   );
 }
+
+const DescriptionListDescriptionStyled = styled(DescriptionListDescription)`
+  opacity: 0.8;
```

Comment:
We might want to use a PF utility class here for styling. https://www.patternfly.org/developer-resources/global-css-variables

Reply:
Changing the opacity makes the text easier to spot and makes it distinct from the label.
Need it to be the same color as before, and still have good contrast for accessibility.
Needs to support dark and light theme.

Many frameworks out there use opacity on text for just this reason.

PF does not have this built in.  I have discussed it with them but it has not landed in PF5.
I do not see any utility classes that would solve this.